src/main/java/com/microsoft/sqlserver/jdbc/Util.java

```diff
@@ -242,10 +242,9 @@ static void writeLong(long value, byte valueBytes[], int offset) {
         Properties p = new Properties();
         String tmpUrl = url;
         String sPrefix = "jdbc:sqlserver://";
-        String result = "";
```

Comment:
Please add tests for the changes you are making.

Reply:
The PR is to change a String variable to a StringBuilder, I don't believe testing will be beneficial.